orchestra/tests/helpers/fixtures.py

```diff
@@ -245,4 +251,52 @@ def setup_models(test_case):
                 task=task,
                 status=assignment_status,
                 assignment_counter=i,
-                in_progress_task_data=task_data)
+                in_progress_task_data=task_data,
+                start_datetime=parse(
+                    '2015-10-12T0{}:00:00+00:00'.format(2 + i)))
+
+
+def task_back_and_forth(test):
+    task = test.tasks['rejected_review']
+
+    test._submit_assignment(
+        test.clients[6], task.id, seconds=35)
+    test._submit_assignment(
+        test.clients[7], task.id, command='reject', seconds=36)
+    test._submit_assignment(
+        test.clients[6], task.id, seconds=37)
+    test._submit_assignment(
+        test.clients[7], task.id, command='accept', seconds=38)
+
+    # Fill out the snapshots for all assignments
```

Comment:
yeah, wrote this comment before I realized the method wasn't inside an individual test. Feel free to leave as is.